internal/component/github_release_source.go

```diff
@@ -38,18 +37,16 @@ func NewGithubReleaseSource(c cargo.ReleaseSourceConfig) *GithubReleaseSource {
 	if c.Type != "" && c.Type != ReleaseSourceTypeGithub {
 		panic(panicMessageWrongReleaseSourceType)
 	}
-	if c.GithubToken == "" {
+	if c.GithubToken == "" { // TODO remove this
```

Comment:
Was this supposed to be removed?

Reply:
No. I was putting in a reminder to remove the panic and replace it with better error handling. The change in this PR had already grown to be too big (with the package version bump and a few other renames, I decided not to fix the error handling yet).

The linked section outlines when a panic is/should be used and in this case we should be returning an error or calling log.Fatal instead of calling panic.

https://google.github.io/styleguide/go/best-practices#program-checks-and-panics